concepts/comparisons/introduction.md

```diff
@@ -1,2 +1,53 @@
-#TODO: Add introduction for this concept.
+# Introduction
 
+A comparison operator in python, also called python relational operator, compares the values of two operands and returns `True` or `False` based on whether the condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, `"!="`, `"is"`
+
+```python
+>>> 7 > 5
+True
+>>> 99 < 100
+True
+>>> 4 < 4
+False
+>>> 4 <= 4
+True
+>>> 1 >= 1
+True
+>>> 5 == 5
+True
+>>> 6 != 6 # not equal to
+False
+>>> 5 != 6
+True

```

Comment:
```suggestion

```

Reply:
The introduction document is displayed to the student _before_ they've completed the exercise.  We try to make them as succinct as we can.  After they complete the exercise, they will be able to see the `about.md` doc.  I think in this circumstance, we  should omit the code examples from the introduction, but have them available to refer back to in the `about.md` after the student has completed the exercise -- if that makes sense?

Reply:
Makes sense to me. Checked a couple of introductions of other concepts and they don't have code sample either. 
To maintain consistency let's remove them